added: CAP fellow posting

abziegler opened the pull request.

abziegler committed:
added: CAP fellow posting

rebeccacremona reviewed (changes requested):
Rewrite of the blurb at /collaborate looks great!

Just a few things it'd be best to do before we merge this in:

1) This pull request includes the changes added in https://github.com/harvard-lil/website-static/pull/76. (Maybe you made the branch "fellow-posting" while you had the branch "cap-contributors" checked out, rather than with "develop" checked out? I can help you rebase the branch, to take care of that.

2) This removes the link to the page built by https://github.com/harvard-lil/website-static/blob/develop/app/collaborate/2017/law-and-tech-fellow.html, but leaves that file up. So, that page will still be live on the web, but we won't link to it anywhere anymore. Is that what you had in mind?

3) The blurb has a link `<a href="{{ site.baseurl }}/collaborate/2017/cap-fellow/"`, but we don't yet have a page there, something like the page referenced in #2 above.

abziegler committed:
added cap-fellow.html

abziegler committed:
fixed: url in cap-fellow page

abziegler commented:
- Rebased successfull (with @rebeccacremona help)
- Fixed URL
- Left reference to other fellow posting ... those pages are still linked out there and no need to pull them all down
